### PR TITLE
Fix lasgun autofire mode not being on at init

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

You had to switch back to standard mode to get autofire

## Changelog
:cl:
fix: Autofire mode in lasgun will be on when you buy the gun
/:cl:

### CODE REVIEW

**Comment by TGCMbetcher** [comment]
Disabler has autofire as well when it shouldnt, might as well fix that while you know the code to do so. I tested the others and its a no. I mean its not like disabler autofire is good for anything one round doesnt even kill a marine.

**Comment by BraveMole** [comment]
> Disabler has autofire as well when it shouldnt, might as well fix that while you know the code to do so. I tested the others and its a no. I mean its not like disabler autofire is good for anything one round doesnt even kill a marine.

Don't think you understand the PR, though i'm not very clear.
Disabler and standard mode both have autofire, it's a feature

**Comment by TGCMbetcher** [comment]
Feature? I was told by an admin autofire was a bug, I guess its been around so log its just normal now. Its actually pretty good so I guess Ill abuse it abit.

File: code/modules/projectiles/guns/energy.dm [hunk 1]
```diff
@@ -477,7 +479,7 @@
 	accuracy_mult_unwielded = 0.5 //Heavy and unwieldy; you don't one hand this.
 	scatter_unwielded = 100 //Heavy and unwieldy; you don't one hand this.
 	damage_falloff_mult = 0.25
-	fire_delay = 3
```
Comment (TiviPlus) [inline code/modules/projectiles/guns/energy.dm:480]: uhhhhh???

Comment (BraveMole) [reply code/modules/projectiles/guns/energy.dm:480]: standard lasgun mode has a fire_delay of 2. And we start in this mode

Comment (TiviPlus) [reply code/modules/projectiles/guns/energy.dm:480]: ok

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**